integration/specs/ColorPicker/colorPicker-1.spec.js

```diff
@@ -0,0 +1,86 @@
+const PageColorPicker = require('../../../src/components/ColorPicker/pageObject');
+const {
+    TAB_KEY,
+    ARROW_UP_KEY,
+    ARROW_LEFT_KEY,
+    ARROW_DOWN_KEY,
+    ARROW_RIGHT_KEY,
+    ALT_KEY,
+} = require('../../constants');
+
+const COLORPICKER = '#picker-color-1';
+
+describe('ColorPicker default variant', () => {
+    beforeAll(() => {
+        browser.url('/#!/ColorPicker/1');
+    });
+    beforeEach(() => {
+        browser.refresh();
+        const component = $(COLORPICKER);
+        component.waitForExist();
+    });
+    it('should focus the right common component when an tap key is pressed', () => {
+        const colorPicker = new PageColorPicker(COLORPICKER);
+        colorPicker.getSaturationPointer().click();
```

Comment:
I think we should create a method for click the saturation pointer directly like: clickSaturationPointer